gdcdictionary/schemas/demographic.yaml

```diff
@@ -254,7 +254,8 @@ properties:
     description: >
       Province or state
     type: string
-    
+
+# This prop originated from the DS4C dataset
```

Comment:
Sorry to be a stickler but can we change all of these to:
"Prop originated from [project_name]"

Reply:
So like this:

Prop originated from DS4C

?